dev_docs/tutorials/building_a_plugin.mdx

```diff
@@ -203,13 +239,15 @@ interface DemoStartPlugins {
 }
 
 export class DemoPlugin {
-  public setup(core: CoreSetup, plugins: DemoSetupPlugins) { [2]
+  public setup(core: CoreSetup, plugins: DemoSetupPlugins) {
```

Comment:
I don't think these are desired changes, we want the `[1]`, `[2]` and `[3]` labels to be on the same line as the thing they're referencing. We could put them behind `//` comments, or change them to `// see [1]`?

Reply:
Oh weird, this somehow auto-formatted. Fixed